### DISCUSSION TITLE
Started discussions

### DISCUSSION THREAD

**Discussion by Nimdy** [Nimdy/Dedicated_Valheim_Server_Script#36] [General]
Just starting a discussion

**Comment by Elliog** [comment]
Hello, great job, works instantly on Azure.

I'm looking for where is located the save of the world ?

**Reply by Elliog** [reply]
I found the path: :

/home/steam/.config/unity3d/IronGate/Valheim

**Reply by Nimdy** [reply]
cool... That that information is found within the wiki

https://github.com/Nimdy/Dedicated_Valheim_Server_Script/wiki

**Comment by atliensarereal** [comment]
This is an awesome project. I was wondering if there is a way to ban / whitelist players? Thanks!

**Reply by Nimdy** [reply]
Yes, you can set yourself as admin and do it from the F5 key or within the folder listing.

You can press F2 and see their STEAMID cut and paste 

/home/steam/.config/unity3d/IronGate/Valheim/ (Files in here for admin/ban/whitelist)

**Comment by stevenmccall** [comment]
Question, how can we access server console on the command line.

Is this running in tmux where you could simply login as the user, and type tmux attach?
What is the user it is running as well?

I am looking to be able to see what is happening on the server while it is happening, people joining, leaving, etc.

**Reply by Nimdy** [reply]
i dont think so... havent explored that yet.

cat /var/log/syslog | grep ZDOID

to monitor live...

tail -f /var/log/syslog | grep ZDOID

will show the names and their IDs
